### PR TITLE
Add xray helper tooling to lambda tools

### PR DESCRIPTION
**Pull request by cmjarreau**
Add xray helper calls to lambda tools; bump eslint

### CODE REVIEW

**Comment by cmjarreau** [comment]
in this branch, linting never flagged the `aws-xray-sdk-core` dependency

**Comment by cmjarreau** [comment]
hmm. graphql test is failing

**Comment by cmjarreau** [comment]
this error is documented here: https://github.com/power-assert-js/empower/issues/28

File: test/graphql-assertions.test.js [hunk 1]
```diff
@@ -76,7 +76,10 @@ test('assertError throws if no error matches the path', async (test) => {
 test('assertError throws if the error does not match the message', async (test) => {
   const query = '{ error }';
   const response = await test.context.graphql(query);
-  test.throws(() => assertError(response, 'error', 'some other message'), `'boom!' == 'some other message'`);
+  const version = process.version.split(/^v(\d+\.\d+)/)[1];
+  if (version.split('.')[0] < 10 && version.split('.')[1] < 9) {
```
[the remark this answers is not in the document]
Comment (cmjarreau) [reply test/graphql-assertions.test.js]: nice fix for this and ava!!

[the remark this answers is not in the document]
Comment (cmjarreau) [reply test/graphql-assertions.test.js]: nice fix for ava!

**Comment by jagoda** [comment]
The fix for ava was merged but I'm not sure when it will be released since they are working on a major version change. I have put in a workaround for lambda-tools until then (see #26) so you should be able to rebase on `master` and clean this up. Let me know if you need any help.

**Pull request merged**